receiver/pluginreceiver/plugin_test.go

```diff
@@ -492,3 +494,44 @@ func TestCheckParameters(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckExtensions(t *testing.T) {
```

Comment:
nit: There's a bit of an unofficial Go standard when testing unexported functions to structure the test name as such `Test_{internal_function_name}`.